### ISSUE TITLE
Lost data

### ISSUE THREAD

**Issue by jctoledot** [resin-io/etcher#2360]
Hello there.
I am a newly user of Linux and tried to do a boot USB ubunutu with Etcher and by mistake I lost all my data (Backup for years) from my eternal HD 4TB.
Now I can see only a ubuntu on it and no more that 700 MB.
How can I go back? 

- **Etcher version:** v1.4.4
- **Operating system and architecture:** OS X
- **Image flashed:** Ubuntu
- **Do you see any meaningful error information in the DevTools?** No

**Comment by lurch** [comment]
I guess this is fairly similar to #2378 ?

**Comment by jhermsmeier** [comment]
Yes, @jctoledot please check #2378, and did Etcher not warn you before flashing a 4TB disk?
